DEVELOPER.md

```diff
@@ -44,13 +45,67 @@ Please run the testing targets locally before you push a PR.
 
 Notes on how to test an existing CR Database.
 
-kubectl exec into a pod
+# Development
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [kustomize](https://kustomize.io/)
+
+The rest of dependencies are packed into Docker images which are executed from the [](Makefile)
+
+### Running the operator in GCP
 
 ```
-cockroach sql
-CREATE TABLE bank.accounts (id INT PRIMARY KEY, balance DECIMAL);
-INSERT INTO bank.accounts VALUES (1, 1000.50);
-SELECT * FROM bank.accounts;
+git clone https://github.com/cockroachdb/cockroach-operator.git
+export CLUSTER=test
+# create a gke cluster
+./hack/create-gke-cluster.sh -c $CLUSTER
+
+# build the image locally and push it to your image repo
```

Comment:
The bazel commands are the only commands that are required to push the operator.  Please test these instructions as I doubt they work.